packages/stylelint-config-react/README.md

```diff
@@ -31,7 +31,7 @@
 With `npm`:
 
 ```sh
-npm install @untile/stylelint-config-react --save-dev
+npm install stylelint @untile/stylelint-config-react --save-dev
```

Comment:
You won't need `stylelint` if you run the last cmd: `npx install-peerdeps --dev @untile/stylelint-config-react`

That should take care of the stylelint dependency. Yarn cmd should also be fixed.

Using the inline install may cause a latest version to be installed instead of the required one.